package-overrides/github/arnaudbreton/angular-youtube@master.json

```diff
@@ -0,0 +1,13 @@
+{
+	"main":"angular-youtube-player-api",
+	"files": ["angular-youtube-player-api.js","build/angular-youtube-player-api.min.js"],
+	"ignore": ["config","scripts","test",".gitignore"],
```

Comment:
You don't need both files and ignore - files by default ignores everything else.